app/screens/channel_info/bindings/bindings.tsx

```diff
@@ -21,11 +23,11 @@ type Props = {
     currentUser: UserProfile;
     appsEnabled: boolean;
     actions: {
-        doAppCall: (call: AppCall, intl: any) => void
+        doAppCall: (call: AppCall, intl: any) => Promise<ActionResult>
     }
 }
 
-const Bindings = (props: Props) => {
+const Bindings: React.FC<Props> = (props: Props) => {
```

Comment:
This was needed to satisfy the `index.ts` file's typescript errors. Maybe an alternative could be done to avoid that, but I don't see any harm in having the appropriate type here.